man/systemd.network.xml

```diff
@@ -2482,12 +2482,15 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
 
       <varlistentry>
         <term><varname>EmitRouter=</varname></term>
-
-        <listitem><para>Similar to the <varname>EmitDNS=</varname>
-        setting described above, this setting configures whether the
-        DHCP lease should contain the router option. The same syntax,
-        propagation semantics and defaults apply as for
-        <varname>EmitDNS=</varname>.</para></listitem>
+        <term><varname>Router=</varname></term>
+
+        <listitem><para>The <varname>EmitRouter=</varname> setting takes a boolean value, and configures
+        whether the DHCP lease should contain the router option. The <varname>Router=</varname> setting
+        takes an IPv4 address. When <varname>Router=</varname> is not specified, then the server address
+        will be used for the router option. When the <varname>EmitRouter=</varname> setting is disabled,
+        the <varname>Router=</varname> setting will be ignored. The <varname>EmitRouter=</varname>
+        setting defaults to true, and the <varname>Router=</varname> setting defaults to unset.
+        </para></listitem>
```

Comment:
I would be explicit here and say that if `Router` is set, it will be used as the value for the router option when `EmitRouter` is enabled.